blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -18,34 +18,41 @@ jobs:
     timeout-minutes: 10
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3<% if (pnpm) { %>
+      - uses: NullVoxPopuli/action-setup-pnpm@v2
```

Comment:
I believe this is the first action we've used in a blueprint that isn't from an official source. 

Should we request it moved into an org we control (as we would do with an addon in the default blueprint). Thoughts @ef4

Reply:
I'm happy to transfer to _some org_ but it doesn't feel like it belongs under ember because it's also used for a bunch of non-ember projects.

I have another organization on GH that I started with @wycats, where, we have an action that wraps the above action here: https://github.com/wyvox/action -- the goal is to get get our repositories set up quickly, and efficiently, without bloating the ci configs and making them hard to manage _while also_ not forcing folks in to all of the features  (for example, the turbo server is only set up if you pass the GH token required for it to store artifacts).

I think it would make the most sense to transfer `NullVoxPopuli/action-setup-pnpm` to `wyvox/action-pnpm`

Thoughts?

(We can also add more folks to the org if that would put people at ease :tada: )

Reply:
I'm going to move the question of where the action to live to the RFC tracking PR and merge this.